vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/accumulator/ChunkedRowAccumulator.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2011-2022 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.sqlclient.impl.accumulator;
+
+import io.vertx.sqlclient.RowIterator;
+
+import java.util.NoSuchElementException;
+import java.util.function.IntUnaryOperator;
+
+public class ChunkedRowAccumulator<T> implements RowAccumulator<T> {
```

Comment:
do we actually use this in our code ?

Reply:
Not yet. I've met with @franz1981 this morning to talk about the last benchmarks results I got. I will run another round of tests this afternoon.

So we can keep it until we make a final decision. But let's not block merging of this PR because of this.

Reply:
sure, but I'm not convinced that the complexity it brings is worth it

Reply:
We'll see what the new results say. It won't be a game changer regarding latency but allocations may be reduced significantly.

Any other comments about the recent commits?

Reply:
all seem fine, we'll rediscuss th chunked accumulaor later then